Runtime/MeshOperations/UV/UvUnwrapping.cs

```diff
@@ -7,12 +7,30 @@ static class UvUnwrapping
         static Vector2 s_TempVector2 = Vector2.zero;
         static readonly List<int> s_IndexBuffer = new List<int>(64);
 
-        internal static void Unwrap(ProBuilderMesh mesh, Face face)
+        internal static void Unwrap(ProBuilderMesh mesh, Face face, Vector3 projection = default(Vector3))
         {
-            Projection.PlanarProject(mesh, face);
+            Projection.PlanarProject(mesh, face, projection != Vector3.zero? projection : Vector3.zero);
             ApplyUVSettings(mesh.texturesInternal, face.distinctIndexesInternal, face.uv);
         }
 
+        /// <summary>
+        /// Copy UVs from faceToCopy to face for the given mesh.
+        /// </summary>
+        /// <param name="mesh">ProbuilderMesh</param>
+        /// <param name="face">face to copy UVs to</param>
+        /// <param name="faceToCopy">face to copy UVs from</param>
```

Comment:
Detail but description and name is a bit confusing to me at first. Also I would kind of of expect the argument to be the other way around source -> destination